### PR TITLE
Mc 6053 usage summary doc

### PR DESCRIPTION
**Pull request by franzpgarcia**

### CODE REVIEW

**Comment by franzpgarcia** [comment]
I am not done with the doc. I opened it too fast sorry XD

File: source/includes/administration/_usage.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+## Usage summary
+
+<!-------------------- LIST USAGE SUMMARY -------------------->
+
+### List usage summary
+
+`GET /usage_summary/organizations/:id`
+
+```shell
+# Retrieve visible environments
```
Comment (simongodard) [inline source/includes/administration/_usage.md]: Wrong comment

**Comment by simongodard** [comment]
Since this page is not public, we will have to commit also in the https://github.com/cloud-ca/api-docs project and publish it

File: source/includes/administration/_usage.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+<!-------------------- LIST USAGE SUMMARY -------------------->
+
+### List usage summary
+
+`GET /usage_summary/organizations/:id`
+
+```shell
+# Retrieve usage summary in JSON
+curl "https://cloudmc_endpoint/v1/usage_summary/organizations/03bc22bd-adc4-46b8-988d-afddc24c0cb5?start_date=2017-05-01&end_date=2017-05-15&format=json" \
```
Comment (pdube) [inline source/includes/administration/_usage.md]: format=json is not required

File: source/includes/administration/_usage.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+   -H "MC-Api-Key: your_api_key"
+
+# Response body example
+```
+```json
+{
+  "data": [{
+    "organizationId": "52fd201e-aa82-4a27-86b3-ea9650a7fb1e",
+    "serviceConnectionId": "beeba736-0451-49b0-8020-8b93ed5abb35",
+    "serviceConnectionPricingId": "e37cc44a-47b6-4a26-81f5-1dbf85433e36",
```
Comment (pdube) [inline source/includes/administration/_usage.md]: Is the SCP id relevant?

**Review by pdube — APPROVED** [review]

File: source/includes/administration/_usage.md (hunk 2 above)
Comment (franzpgarcia) [inline source/includes/administration/_usage.md]: I changed this to have a json example and CSV example

File: source/includes/administration/_usage.md (hunk 3 above)
Comment (franzpgarcia) [inline source/includes/administration/_usage.md]: No it's an example.

Comment (pdube) [inline source/includes/administration/_usage.md]: I mean, why are we returning it? Can end users query service connection pricings?

Comment (franzpgarcia) [inline source/includes/administration/_usage.md]: It was already there and it was in the acceptance criteria. We have the information so why not :S

**Pull request merged**